diff --git a/guiclient/main.cpp b/guiclient/main.cpp
index 93405a0..5889d7a 100644
--- a/guiclient/main.cpp
+++ b/guiclient/main.cpp
@@ -301,7 +301,7 @@ int main(int argc, char *argv[])
                " WHERE pkghead_name IN ('xtmfg');" )
           << editionDesc( "Distribution",       ":/images/splashDistEdition.png",        true,
                "SELECT fetchMetricText('Application') = 'Standard';" )
-          << editionDesc( "PostBooks",      ":/images/splashPostBooks.png",        true,
+          << editionDesc( "PostBooks",      ":/images/splashPostBooks.png",        false,
                "SELECT fetchMetricText('Application') = 'PostBooks';" )
   ;
 
